### DISCUSSION TITLE
Class Specific MOT Scores

### DISCUSSION THREAD

**Discussion by ocetintas** [bdd100k/bdd100k#232] [Q&A] [answered]
We are looking for a way to evaluate our tracker only for the **pedestrian category** on **BDD100K Multiple Object Tracking**. However, the leaderboard on eval.ai does not list the **class-specific scores**. 

**1)** Is it possible to access our class-specific scores (e.g. MOTA and IDF1) once we make a submission?

**2)** Is it possible to check the class-specific scores of previous submissions (e.g. QDTrack, ByteTrack) so that we can compare our method with them?

**Comment by siyuanliii** [comment, marked as answer]
Hi, we are currently updating the benchmark. If you need detailed results now, maybe you could send your final results to us and we will do the evaluation for you. We could also provide the breakdown of qdtrack results.

**Reply by ocetintas** [reply]
Hi, thank you for your response. Most likely we will need class-specific scores within the next two weeks. In this case, it would be great if you can help us get the class-specific scores. Should we reach you via email?

**Reply by siyuanliii** [reply]
Yes, please share with us your results in the scalabel format. I will do the test for you and send your results back via email.
